components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/APIImportExportConstants.java

```diff
@@ -136,4 +136,5 @@
         // To identify thumbnail icons
         fileExtensionMapping.put("application/json", "json");
     }
+
```

Comment:
Shall we remove this new line ?

Reply:
c1f846270025eb67fcd8e92c102a8539e1e699a3